src/components/NetworkDetails.js

```diff
@@ -51,7 +51,7 @@ export const NetworkDetails = ({
             </span>
         </p>
         <p className="details-data-container">
-          <span className="details-label">Remaining Daily {action} Quota</span>
+          <span className="details-label">Remaining Daily {currency} Quota</span>
```

Comment:
Please confirm that limits statistic will not be applicable to foreign side in case of `erc-to-erc` mode since we have no such limits in this mode.

Reply:
@akolotov On foreign side on erc20-erc20 mode, checks for remaining daily quota, minimum amount and maximum amount per  transaction are not applicable. Is that ok?

https://github.com/poanetwork/bridge-ui/blob/8dab5d9ee356e63b3ef324ebcbbdc2b0b516d640/src/components/Bridge.js#L117-L128

Reply:
Thanks for pointing this! I pushed a fix to hide that information on Foreign side